### PR TITLE
Update affiliation for @nknize.

### PR DESCRIPTION
**Pull request by dblock**
### Description

Updated affiliation for @nknize in MAINTAINERS.md.

@nknize lmk if you prefer to be moved to Emeritus or would like a different affiliation since we haven't seen you here in a while.
 
By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Review by peternied — APPROVED** [review]

**Comment by nknize** [comment]
> @nknize lmk if you prefer to be moved to Emeritus or would like a different affiliation since we haven't seen you here in a while.

Thank you for asking. Title aside, my time spent building Lucenia will likely leave little left to donate code here in the near term, but I'm happy to contribute in other ways if ever asked.  If that warrants stripping my commit bit and maintainer access level, well I leave that up to y'all. Personally, I've always been a proponent of "merit never expires" but I died on that hill a long time ago.

**Comment by dblock** [comment]
@nknize No need to remove your r/w privileges, it's not a problem at all. I'll merge this PR as is.

**Pull request merged**
